### PR TITLE
Download scan results

### PR DESCRIPTION
**Pull request by caggle**
I am considering two options:
1.  Expose an endpoint which accepts a host, searches the S3 bucket, gets all the output for the host, compiles together (tar.gz) and returns in one file
OR
2. Expose an endpoint which accepts a host, searches the S3 bucket, gets all the output, compiles together in a ZIP/tar.gz and returns a presigned URL as a response which the user should just puts in a browser or curls.

I am not sold on which one yet...they probably will require similar effort as most of what's required is the same.

### CODE REVIEW

**Comment by caggle** [comment]
I went with option 1, frankly because it was easier.

Having said this, now that option 1 is implemented, it's easier to implement option 2 as well. 

A down-side of option 1 is it will not work properly with a browser (as it is a POST request and because of the `Accept` header weirdness with AWS API gateway. But I wrote a new "client" under examples to achieve this.

Option 2 would work with browsers as in theory it would return a pre-signed URL that can be fetched with a GET without authentication.

I still think I want to implement option 2 eventually, it'd be neat.

@claudijd unless you feel strongly about option 2, I think this is ready for review.

File: README.md [hunk 1]
```diff
@@ -63,13 +64,11 @@ _**Note:** UDP port scans are not supported as Lamdba functions can not run as r
...
 
-10. If you have no CloudFormation errors and if you see `Service Information` listing your lambda functions/endpoints, you are good to go.
+10. If you have no serverless/CloudFormation errors and if you see `Service Information` listing your lambda functions/endpoints, you are good to go.
 
 ## Examples
 
-- Once deployed properly, you can kick of an ondemand scan on a host, using: `API_PROFILE=<YOUR-AWS-PROFILE/ROLE> python3 examples/ondemand_tasker.py`. Alternatively, you can hard-code your AWS profile in a variable in the code for examples.
-  
-  Note: You must provide an AWS profile/role if you'd like to use them as clients. This is required in order to programmatically fetch the API gateway URL details for your vautomator-serverless instance, as well as the API gateway key which currently protects the REST APIs.
+- Once deployed properly, you can kick of an ondemand scan on a host, using: `AWS_PROFILE=<YOUR-AWS-PROFILE/ROLE> python3 examples/ondemand_tasker.py`. Alternatively, you can hard-code your AWS profile in a variable in the code for examples.
```
Comment (claudijd) [inline README.md]: s/off/of/

Comment (caggle) [reply README.md]: Fixed.

**Comment by caggle** [comment]
> Notice a typo in the readme, otherwise, great job on this!

Thanks!

**Pull request merged**